tests/test_scheduled_job.py

```diff
@@ -175,4 +176,4 @@ def test_job_trigger(self):
         resp = self.client.post('/api/scheduled_job/{}/trigger'.format(job.id))
         assert resp.status_code == 200
 
-        db.session.close_all()
+        session.close_all_sessions()
```

Comment:
I would expect `db.session` is referencing the same symbol.  Did it not work with `db.session.close_all_sessions`?

Reply:
db.session.close_all_sessions does not work because db.session is a Session object (sqlalchemy.orm.session.Session) which does not have close_all_sessions method.
The close_all_sessions method is in at sqlalchemy.orm.session.close_all_sessions.